### ISSUE TITLE
Start setup error after Sign in with Google: Unable to authenticate Site Kit. Check your client configuration is in the correct JSON format

### ISSUE THREAD

**Issue by kospl** [google/site-kit-wp#768]
## Bug Description

Plugin setup cannot be completed after Sign in with Google because of an error with config.

## Steps to reproduce

1. Install plugin
2. Go to "Start Setup"
3. On Google's page select "Sign in with Google"
4. Allow access to data
5. It redirects back to /wp-admin with error:
> Unable to authenticate Site Kit. Check your client configuration is in the correct JSON format.

## Screenshots

![image](https://user-images.githubusercontent.com/26386660/68073289-0f605e00-fdc1-11e9-901f-8ca867ee4ba9.png)

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

## Changelog entry

*

**Comment by Juke29** [comment]
Same here. But first I was stuck on sitekit web page. Process stopped at step 2 when clicking on the button "Allow", button is disabled and nothing more happens: 

> 1) You're a verified owner of mysite.com
> 2) Allow mysite.com to access Google Account data
> Now that you're a verified owner of mysite.com, your site wants to access your Google Account data to display metrics on the Site Kit dashboard.
> 
> You can view or remove access for mysite.com in your Site Kit dashboard at any time.
> 
> Allow Cancel

Error in console : 
> [amp-form] Failed to parse response JSON: SyntaxError: Unexpected end of JSON

Refreshed page, error, had to go back to wordpress and try to Reset Site Kit but that does not do anything. So I tried to re Start setup and got the same error "Unable to authenticate Site Kit. Check your client configuration is in the correct JSON format.". Tried to Start setup again, logged in Google, allowed access again, and this time it skipped to step 3 "add site" and after clicking: button "Allow" disabled, nothing happens again... refreshed page, and got congratulations go to my dashboard. It works now. I have no idea why. I just kept trying to redo every thing the same way, 3 times.

**Comment by sorburu** [comment]
Same problem here... :(

**Comment by lunaluna** [comment]
I have the same problem.
1. Sign in with google
2. Log in to google
3. Allow access to data
4. Error 'Unable to authenticate Site Kit.Check your client configuration is in the correct JSON format.' in Dashboard

Language: Japanese
Version: WordPress 5.3.1-alpha-46728
Server: X-server

**Comment by sorburu** [comment]
It's working for me since yesterday 1.0.4 version :)

**Comment by kospl** [comment]
Still an issue with 1.0.4.

**Comment by erlupacchiotto** [comment]
same problem... 👎

**Comment by marty80** [comment]
Same issue. Please fix. Nothing in console.

**Comment by hopindigital** [comment]
> ## Bug Description
> Plugin setup cannot be completed after Sign in with Google because of an error with config.
> 
> ## Steps to reproduce
> 1. Install plugin
> 2. Go to "Start Setup"
> 3. On Google's page select "Sign in with Google"
> 4. Allow access to data
> 5. It redirects back to /wp-admin with error:
> 
> > Unable to authenticate Site Kit. Check your client configuration is in the correct JSON format.
> 
> ## Screenshots
> ![image](https://user-images.githubusercontent.com/26386660/68073289-0f605e00-fdc1-11e9-901f-8ca867ee4ba9.png)
> 
> _Do not alter or remove anything below. The following sections will be managed by moderators only._
> 
> ## Acceptance criteria
> * 
> 
> ## Implementation Brief
> * 
> 
> ## Changelog entry
> *

Just Turn off the Security Plugins and it will work properly.

Regards
`[HOP IN DIGITAL](www.hopindigital.com)`

**Comment by dote78** [comment]
> Just Turn off the Security Plugins and it will work properly.

Not for me. I deactivated the only security plugin I have, "All in one WP Security", and it fails in the same way, with same message. But, given the hint, I deactivated some others also, and it completed. I suspect it was "Coming soon by Seedprod", because I had the mode enabled (site is being built atm), wich of course could interfere with any response requisites from SiteKit. My bad!! T_T
